src/zed_cpu_ros.cpp

```diff
@@ -295,25 +298,9 @@ class ZedCameraROS {
 		right_info.K[5] = r_cy;
 		right_info.K[8] = 1.0;
 
-		// rectification matrix
-		// Rl = R_rect, R_r = R * R_rect
-		// since R is identity, Rl = Rr;
 		left_info.R.fill(0.0);
 		right_info.R.fill(0.0);
-		cv::Mat rvec = (cv::Mat_<double>(3, 1) << rx, ry, rz);
```

Comment:
according to
https://github.com/stereolabs/zed-ros-wrapper/issues/39

the rx, ry, rz is RX, RY and CV.
It says the image published by SDK is rectified, but doesn't mean from the USB VideoCapture is rectified, since ZED doesn't have any onboard processor.
So this rx ry rz is useful. Please recover this.